app/models/hostgroup.rb

```diff
@@ -194,22 +194,10 @@ def root_pass
 
   # Clone the hostgroup
   def clone(name = "")
-    new = self.dup
-    new.name = name
-    new.title = name
-    new.puppetclasses = puppetclasses
-    new.locations     = locations
-    new.organizations = organizations
+    new = self.deep_clone(:include => [:group_parameters, :locations, :organizations, :hostgroup_classes, :lookup_values],
```

Comment:
I guess the call here for `set_lookup_value_matcher` was never needed, as it'd happen upon save any way. @unorthodoxgeek correct?